### ISSUE TITLE
Re-add antiminge code to Chaplain

### ISSUE THREAD

**Issue by SpadesNeil** [TheReeplex/PrisonStation#21]
> 2:02 AM - ·:RX:· Deadi: There's a literal function for if the chaplain sets something as homosexuality?
> 2:02 AM - Spades: Yes
> 2:02 AM - Spades: lol
> 2:02 AM - ·:RX:· Deadi: LOL
> 2:02 AM - ·:RX:· Deadi: I didn't know that
> 2:02 AM - Spades: it was originally like
> 2:02 AM - Spades: a dev's way of saying "fuck you"
> 2:02 AM - Spades: like
> 2:02 AM - Spades: if I set the religion to "faggot"
> 2:02 AM - Spades: I used to get brain damage
> 2:03 AM - ·:RX:· Deadi: Well yeah cause people are idiots or like to bait
> 2:03 AM - ·:RX:· Deadi: Heavily
> 2:03 AM - Spades: I'd be happy to re-add that feature.
> 2:03 AM - ·:RX:· Deadi: Oh please do
> 2:03 AM - ·:RX:· Deadi: I love when the script works against asshats
> 2:04 AM - Spades: I should set it so
> 2:04 AM - Spades: if they set religion to "Jihad"
> 2:04 AM - Spades: they gib
> 2:05 AM - ·:RX:· Deadi: YES
> 2:05 AM - ·:RX:· Deadi: I'D LOVE THAT
> 2:05 AM - ·:RX:· Deadi: Or pretty much any mention of it
> 2:05 AM - ·:RX:· Deadi: Like "Jihad" "Jihadi" "ISIS" "Taliban" "Al Queda" etc
> 2:05 AM - ·:RX:· Deadi: Or "Allahu Akbar" "Allahu Ackbar"
> 2:05 AM - ·:RX:· Deadi: The usual

tl;dr hilariously punish asshats for abusing the ability to name their own religion
